instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-testing/src/main/groovy/JaxRsAnnotationsInstrumentationTest.groovy

```diff
@@ -32,14 +33,15 @@ abstract class JaxRsAnnotationsInstrumentationTest extends AgentTestRunner {
     assertTraces(1) {
       trace(0, 1) {
         span(0) {
-          name "POST /a"
+          name "/a"
           attributes {
           }
         }
       }
     }
   }
 
+  @Unroll
```

Comment:
I think we have global unroll, so this is not needed

Reply:
AFAIR only HTTP client & server base test classes have `@Unroll` on them, every test that does not extend one of them still needs to use `@Unroll`